### DISCUSSION TITLE
Podanie o dodanie SHT-30

### DISCUSSION THREAD

**Discussion by tnowak** [hackerspace-silesia/Smogomierz#120] [Ideas]
Cześć!

Fajnie że Sensirion SPS-30 został dodany.
Czy jest szansa na dodanie też czujnika temperatury i wilgotności Sensiriona SHT-30?
Ostatnio natknąłem się na taki fajny w obudowie pyłoszczelnej:
https://botland.com.pl/czujniki-pogodowe/14969-sht-30-czujnik-temperatury-i-wilgotnosci-odporny-na-warunki-atmosferyczne-i2c-1m-adafruit-4099.html
i myślę że fajnie, gdyby tez był obsługiwany

Pozdro!

**Comment by bfaliszek** [comment]
Trochę drogie rozwiązanie jak na SHT-30, zwłaszcza, że widziałem je za ~40zł - https://allegro.pl/oferta/czujnik-temperatury-i-wilgotnosci-gleby-sht30-10092884720 

Postaram się dodać obsługę SHT-30 przy kolejnej aktualizacji

**Comment by tnowak** [comment]
Hej i jak tam Twoje możliwości czasowe?
Bo sprzęt leży już na półce i czeka na testy

**Comment by tnowak** [comment]
Cześć, tylko przypominam się, że nadal SHT-30 czeka na półce :)
